sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -427,8 +433,13 @@ SUBROUTINE MDLFLD
 !
            DO J=JSTA,JEND
            DO I=1,IM
+           IF(P1D(I,J)<spval.and.T1D(I,J)<spval.and.Q1D(I,J)<spval)THEN
```

Comment:
@JesseMeng-NOAA 
Shouldn't it be
IF(C1D(I,J)<spval.andFI1D.(I,J)<spval)THEN

Reply:
will do

Reply:
done